File: app/src/main/java/emperatriz/riverflood/model/Link.java
package emperatriz.riverflood.model;

/**
 * Created by ramon on 24/04/2017.
 */

public class Link {

    public static final int ESP=0, ENG=1, POR=2, ITA=3, RU=4, DESCONOCIDO=5, GER=6, UCR=7, SUE=8, CRO=9, LET=10, FRA=11;
    public static final int FPS50=0, FPS25=1, NOFPS=2;
    public static final int HD1080=0, HD720=1, SD576=2, NORES=4;

    public String url, acestream;
    public int idioma, kbps, fps, resolution;
}

File: app/src/main/java/emperatriz/riverflood/model/Link.kt
package emperatriz.riverflood.model

/**
 * Created by ramon on 24/04/2017.
 */

class Link {

    var url: String? = null
    var acestream: String? = null
    var idioma: Int = 0
    var kbps: Int = 0
    var fps: Int = 0
    var resolution: Int = 0

    companion object {

        val ESP = 0
        val ENG = 1
        val POR = 2
        val ITA = 3
        val RU = 4
        val DESCONOCIDO = 5
        val GER = 6
        val UCR = 7
        val SUE = 8
        val CRO = 9
        val LET = 10
        val FRA = 11
        val FPS50 = 0
        val FPS25 = 1
        val NOFPS = 2
        val HD1080 = 0
        val HD720 = 1
        val SD576 = 2
        val NORES = 4
    }
}
